decryptor/postgresql/utils.go

```diff
@@ -176,6 +189,142 @@ func (p *BindPacket) Zeroize() {
 	}
 }
 
+// GetParameters extracts statement parameters from Bind packet.
+func (p *BindPacket) GetParameters() ([]base.BoundValue, error) {
+	values := make([]base.BoundValue, len(p.paramValues))
+	for i := range values {
+		encoding, err := p.parameterEncoding(i)
```

Comment:
can't get used to golang's forloops (if one argument and iterate array then it is index, two args is index and value. but with maps one arg is key)... 
I thought that `i` is element of empty array, not index and try to understand why we pass empty element to `parameterEncoding`. Then I tried to understand why we pass just index to method named like field and expect result *(

can we rename `parameterEncoding` to `getParameterEncodingByIndex` or smth like that?